fluss-examples/docker-compose.yml

```diff
@@ -0,0 +1,132 @@
+#  /*
+#    * Licensed to the Apache Software Foundation (ASF) under one or more
+#    * contributor license agreements.  See the NOTICE file distributed with
+#    * this work for additional information regarding copyright ownership.
+#    * The ASF licenses this file to You under the Apache License, Version 2.0
+#    * (the "License"); you may not use this file except in compliance with
+#    * the License.  You may obtain a copy of the License at
+#    *
+#    *    http://www.apache.org/licenses/LICENSE-2.0
+#    *
+#    * Unless required by applicable law or agreed to in writing, software
+#    * distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+#    */
+
+# Zoopeeker service
+
+services:
+  zoo1:
+    image: zookeeper:3.9.2
+    restart: always
+    hostname: zoo1
+    ports:
+      - 2181:2181
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_SERVERS: server.1=zoo1:2888:3888;2181 server.2=zoo2:2888:3888;2181 server.3=zoo3:2888:3888;2181
+    networks:
+      - fluss-net
+
+  zoo2:
+    image: zookeeper:3.9.2
+    restart: always
+    hostname: zoo2
+    ports:
+      - 2182:2181
+    environment:
+      ZOO_MY_ID: 2
+      ZOO_SERVERS: server.1=zoo1:2888:3888;2181 server.2=zoo2:2888:3888;2181 server.3=zoo3:2888:3888;2181
+    networks:
+      - fluss-net
+
+  zoo3:
+    image: zookeeper:3.9.2
+    restart: always
+    hostname: zoo3
+    ports:
+      - 2183:2181
+    environment:
+      ZOO_MY_ID: 3
+      ZOO_SERVERS: server.1=zoo1:2888:3888;2181 server.2=zoo2:2888:3888;2181 server.3=zoo3:2888:3888;2181
+    networks:
+      - fluss-net
+
+  # Fluss services
+
+  coordinator:
+    image: fluss/fluss:0.7.0
+    hostname: coordinator-server
+    container_name: coordinator-server
+    command: [ "coordinatorServer" ]
+    depends_on:
+      - zoo1
+      - zoo2
+      - zoo3
+    ports:
+      - "9123:9123"
+    environment:
+      FLUSS_PROPERTIES: |
+        zookeeper.address: zoo1:2181,zoo2:2181,zoo3:2181
+        bind.listeners: INTERNAL://coordinator-server:0, CLIENT://coordinator-server:9123
+        advertised.listeners: CLIENT://localhost:9123
+        internal.listener.name: INTERNAL
+    networks:
+      - fluss-net
+
+  tablet-server-0:
```

Comment:
I think it might be better to have 3-tablet servers instead of 3 zk server, to simulate a distributed environment, so its more realistic. WDYT? You can find one such file [here](https://github.com/ververica/ververica-fluss-examples/blob/main/fluss-java-client/docker-compose.yaml)